packages/compiler-cli/ngcc/test/integration/ngcc_spec.ts

```diff
@@ -348,6 +348,84 @@ runInEachFileSystem(() => {
          });
     });
 
+    it(`should be able to detect synthesized constructors in ES5 with downlevelIteration enabled (imported helpers)`,
+       () => {
+         setupAngularCoreEsm5();
+         compileIntoApf(
+             'test-package', {
+               '/index.ts': `
+                import {Injectable} from '@angular/core';
+
+                @Injectable()
+                export class Base {}
+
+                @Injectable()
+                export class SubClass extends Base {
+                  constructor() {
+                    // Note: mimic the situation where TS is first emitted into ES2015, resulting
+                    // in the spread super call below, and then downleveled into ES5 using the
+                    // "downlevelIteration" option.
+                    super(...arguments);
+                    this.foo = 'bar';
+                  }
+                }
+              `,
+             },
+             {importHelpers: true, noEmitHelpers: true, downlevelIteration: true});
+
+         mainNgcc({
+           basePath: '/node_modules',
+           targetEntryPointPath: 'test-package',
+           propertiesToConsider: ['esm5'],
+         });
+
+         const jsContents = fs.readFile(_(`/node_modules/test-package/esm5/src/index.js`));
+         // Verify that the ES5 bundles does contain the expected downleveling syntax.
+         expect(jsContents).toContain('__spreadArray([], __read(arguments))');
+         expect(jsContents)
+             .toContain(
+                 'var ɵSubClass_BaseFactory = /*@__PURE__*/ ɵngcc0.ɵɵgetInheritedFactory(SubClass);');
+       });
+
+    it(`should be able to detect synthesized constructors in ES5 with downlevelIteration enabled (emitted helpers)`,
+       () => {
+         setupAngularCoreEsm5();
+         compileIntoApf(
+             'test-package', {
+               '/index.ts': `
+                import {Injectable} from '@angular/core';
+
+                @Injectable()
+                export class Base {}
+
+                @Injectable()
+                export class SubClass extends Base {
+                  constructor() {
+                    // Note: mimic the situation where TS is first emitted into ES2015, resulting
+                    // in the spread super call below, and then downleveled into ES5 using the
+                    // "downlevelIteration" option.
+                    super(...arguments);
+                    this.foo = 'bar';
+                  }
+                }
+              `,
+             },
+             {importHelpers: false, noEmitHelpers: false, downlevelIteration: true});
+
+         mainNgcc({
+           basePath: '/node_modules',
+           targetEntryPointPath: 'test-package',
+           propertiesToConsider: ['esm5'],
+         });
+
+         const jsContents = fs.readFile(_(`/node_modules/test-package/esm5/src/index.js`));
+         // Verify that the ES5 bundles does contain the expected downleveling syntax.
```

Comment:
Note: I'll fix the typo here.
```suggestion
         // Verify that the ES5 bundle does contain the expected downleveling syntax.
```